terasoluna-tourreservation-selenium/src/test/java/org/terasoluna/tourreservation/tourreserve/selenium/TokenCheckErrorTest.java

```diff
@@ -261,7 +264,11 @@ public void testReserveUpdateToken() {
                 "_TRANSACTION_TOKEN")).clear();
 
         // change reservation
-        driver.findElement(By.id("changeBtn")).click();
+        jse.executeScript("document.getElementById('changeBtn').click();");
+
+        Wait<WebDriver> wait = new WebDriverWait(driver, 4, 2000);
```

Comment:
- [ ] You should use `selenium.webDriverWait` & `selenium.webDriverSleepWait` properties.